engine/bots.go

```diff
@@ -17,7 +17,6 @@ type Bot struct {
 	Description string `db:"description"`
 	Username    string `db:"username"`
 	Password    string `db:"password"`
-	CreatedBy   string `db:"created_by"`
```

Comment:
Are we sure we don't have a need for it? This was meant to be used as a way to distinguish what bots were created by us & supplied manually and bots that would have been registered by neurobot on the fly.